ghost/core/core/shared/url-utils.js

```diff
@@ -14,3 +13,6 @@ const urlUtils = new UrlUtils({
 
 module.exports = urlUtils;
 module.exports.BASE_API_PATH = BASE_API_PATH;
+module.exports.STATIC_IMAGE_URL_PREFIX = 'content/images';
```

Comment:
This is already exposed on the `UrlUtils` instance, but I've added it for consistency with the other two.

All three of these `module.exports` assignments for the constants will be removed once `UrlUtils` has been updated to expose the values (I'm currently converting it to TS)